### DISCUSSION TITLE
Building the solution

### DISCUSSION THREAD

**Discussion by eugenekop** [PowerShell/PowerShell#18533] [Q&A]
How do I fix these errors when I'm building the solution?

![image](https://user-images.githubusercontent.com/15242950/201299893-f84a0c77-1cc6-4aa9-9650-c18e458ad920.png)

**Comment by ALIENQuake** [comment]
[Official docs: https://github.com/PowerShell/PowerShell/blob/master/docs/building/windows-core.md](https://github.com/PowerShell/PowerShell/blob/master/docs/building/windows-core.md#building-in-visual-studio)

https://github.com/PowerShell/PowerShell/issues/3400

I think that you are required to use the`.\build.psm1` module with `Start-PSBuild` command as of today.
